app/bundles/CategoryBundle/Translations/en_US/messages.ini

```
@@ -9,8 +9,8 @@ mautic.category.header.new="New Category"
 mautic.category.menu.index="Categories"
 mautic.category.noresults.tip="Categories help organize and group items together. In many of the filters, you can type category:{alias} (replace {alias} with the real deal) to show the items within a specific category."
 mautic.category.contentblock.heading="Bring order to your resources"
-mautic.category.contentblock.subheading="Keep your marketing campaigns, emails, assets, forms, and other elements neatly organized."
-mautic.category.contentblock.copy="Categories help you group related items together, making them easier to find, manage, and report on - like folders for your marketing initiatives."
+mautic.category.contentblock.subheading="Add clarity and classification to your campaigns, emails, assets, and other elements."
+mautic.category.contentblock.copy="Categories act as high-level labels that help you identify related items, making them easier to differentiate and report on."
```

```suggestion
mautic.category.contentblock.copy="Categories act as high-level labels that help you identify related items, making them easier to differentiate and report on. Contacts can also select their interests by category in the preference centre."
```

I think this description is missing a key functionality of categories, that of using them in preference centres to group interests. Not everyone uses them in this way, but quite a lot of folks do.

---

This is one issue mentioned by John, categories are used for many random things
I'd say this helps differentiating both